appender_test.go

```diff
@@ -58,6 +57,8 @@ func TestAppender(t *testing.T) {
 	var bytesTotal int64
 	var bytesUncompressed int64
 	client := docappendertest.NewMockElasticsearchClient(t, func(w http.ResponseWriter, r *http.Request) {
+		require.Len(t, r.URL.Query(), 1)
```

Comment:
it may sound stupid but I prefer we have a test that Unset, True, False are propagated correctly to the http request